src/serial_bridge_utils.cpp

```diff
@@ -149,3 +149,44 @@ string serial_bridge_utils::error_ret_json_from_code(int code, optional<string>
 	//
 	return ret_json_from_root(root);
 }
+
+string serial_bridge_utils::decode_base64(const string &input, string &out) {
```

Comment:
Is this part of the original `serial_bridge_utils`? or is it added here by us?

Reply:
It was added by me. We forked mymonero for a long time, and not only this function is added by us, extract-utxo, create-request, convert-to-json we're also added before.

Reply:
Do you mean only moving decode_base64 to another file?

Reply:
Yes.